products/views.py

```diff
@@ -0,0 +1,33 @@
+import json
+from django.http     import JsonResponse
+from django.views    import View
+
+from products.models import Product, ProductImage, ProductOption, ProductDescription, Ingredient, Tag
+class ProductDetailView(View):
+    def get(self, request, product_option_id):
+        product_option = ProductOption.objects.get(id=product_option_id)
```

Comment:
product_option_id 를 받기 때문에 ProductOption 에서 시작하시는 것 같습니다!
프론트로부터 넘겨받는 값은 실제로 product_id 이기 때문에 Product 로부터 시작해보세요!
```suggestion
        products = Product.objects.get(id=product_id)
```

Reply:
위 답변을 참고하여 피드백 부탁드립니다!